### DISCUSSION TITLE
Logo usage in other projects + licence

### DISCUSSION THREAD

**Discussion by marschwar** [detekt/detekt#3752] [Q&A]
I was thinking about adding the logo to the Jenkins Plugin [warnings-ng](https://github.com/marschwar/warnings-ng-plugin).

1. Is that something we want to do?
2. They only support png so I had to convert the logo. Then I realized that they have a Page where they list [icon licences](https://github.com/marschwar/warnings-ng-plugin/blob/master/plugin/src/main/webapp/icons/LICENSE.txt) and I was wondering what to put there.

Before submitting a PR there I wanted to hear your opinions on that.

**Comment by cortinico** [comment]
_I'm not a lawyer_

> Is that something we want to do?

I believe we're good to go 👍 As a rule of thumb the logo is protected by the project License ([Apache 2.0](https://opensource.org/licenses/Apache-2.0) - That has a specific trademark license).

I'll discuss with @payathedo if she wants to release the asset as a separate license (perhaps a CC-like license).
